receiver/splunkhecreceiver/config.go

```diff
@@ -33,9 +32,8 @@ type Config struct {
 	confighttp.HTTPServerSettings `mapstructure:",squash"` // squash ensures fields are correctly decoded in embedded struct
 
 	splunk.AccessTokenPassthroughConfig `mapstructure:",squash"`
-	// Path we will listen on, defaults to `*` (anything matches)
-	Path     string `mapstructure:"path"`
-	pathGlob glob.Glob
+	// RawPath for raw data collection, default is '/services/collector/raw'
+	RawPath string `mapstructure:"raw_path"`
```

Comment:
Is this a breaking change? How do we handle this? At least changelog entry. Is the other config not useful? What is the behavior listen on everything with normal except raw_path where does the raw?

Reply:
adding changelog entry. No, path is not helpful as discussed earlier in this PR here: https://github.com/open-telemetry/opentelemetry-collector-contrib/pull/4951#discussion_r699861877

Reply:
@atoulme you should probably at least make sure the `path` config is still accepted and just ignored so it won't break anybody's configs.

Reply:
missed that. OK I can add that.